README.md

```diff
@@ -91,6 +91,21 @@ import { reset } from 'ember-window-mock';
 
 This function should be called before all tests that depend on the window mock, preferably in the `beforeEach` hook. See below for some examples!
 
+### Resetting the state in with QUnit 2.0
```

Comment:
AFAIK the new API is not really related to QUnit 2.0, rather to the RFC, aka the "new testing APIs".

Reply:
I renamed the README headline. Not sure if it's fitting enough.